Cargo.toml

```diff
@@ -181,6 +181,7 @@ aws-sdk-cloudwatchlogs = { version = "1.3.0", default-features = false, features
 aws-sdk-elasticsearch = { version = "1.3.0", default-features = false, features = ["behavior-version-latest"], optional = true }
 aws-sdk-firehose = { version = "1.3.0", default-features = false, features = ["behavior-version-latest"], optional = true }
 aws-sdk-kinesis = { version = "1.3.0", default-features = false, features = ["behavior-version-latest"], optional = true }
+aws-sdk-sts = { version = "1.3.0", default-features = false, features = ["behavior-version-latest"], optional = true }
```

Comment:
I didn't see anything in this PR that is using `aws_sdk_sds` directly. Is this added just to add the feature flag, or is this a remnant of an intermediate step?

Reply:
It needs the feature flag, and if I recall the error only occurs at runtime when you select a specific combination of authentication mechanisms. I'll spend a moment trying to replicate it - but for safety I think it's probably worth keeping it there.